src/components/ListBackend.tsx

```diff
@@ -0,0 +1,4 @@
+import { collection, addDoc, updateDoc, deleteDoc, doc, query, CollectionReference } from 'firebase/firestore';
+import { useCollection } from 'react-firebase-hooks/firestore';
+
+import { db } from '../../firebase.config';
```

Comment:
In React world isolation of the 'backend' part of the component from the actual UI is a good and encouraged thing. But instead of moving it to the component you usually want to move it to a custom hooks. I know it's a new thing for you guys. Here's the official guidebook on the topic: https://react.dev/learn/reusing-logic-with-custom-hooks
Also, I'll try to add an example in the following PR